utils.go

```diff
@@ -27,3 +29,13 @@ func waitForAliveVMM(ctx context.Context, client *Client) error {
 		}
 	}
 }
+
+// envValueOrDefaultInt check if env value exists and returns it or returns default value
+// provided as a second param to this function
+func envValueOrDefaultInt(envName string, def int) int {
```

Comment:
I don't really love this function name, but I guess it's descriptive enough and I can't really think of anything better right now.